content/docs/administration/permissions.md

```diff
@@ -24,41 +24,43 @@ The following table lists the area and name of the permissions in the central se
+| Permissions (2)      | Patients              | View patients                       | Can view Patients                                                                                            |
+|                      |                       | Add Patients                        | Can maintain Patients                                                                                        |
+|                      |                       | Edit Patient Details                | Can maintain Patients - if either add or edit permission is given the user can edit                          |
+| Permissions (3)      | Admin                 | View log                            | Able to view the logs, these show as a tab on many windows                                                   |
+| Permissions (3)      | Requisitions          | View requisitions                   | Can view Requisitions                                                                                        |
+|                      |                       | Create and edit requisitions        | Can maintain Requisitions                                                                                    |
+| omSupply Permissions |                       | Confirm Internal Order sent         | Can confirm Internal Orders as sent                                                                          |
+| Permissions (3)      | Stocktakes            | Create Stocktake                    | Can create Stocktakes                                                                                        |
+|                      |                       | Delete Stocktake                    | Can delete Stocktakes                                                                                        |
+|                      |                       | Add Stocktake lines                 | Can add Stocktake lines                                                                                      |
+|                      |                       | Edit Stocktake lines                | Can edit Stocktake lines                                                                                     |
+| Permissions (3)      | Vaccines              | View sensor details                 | Can view sensor details                                                                                      |
+|                      |                       | Edit sensor location                | Can edit sensor location                                                                                     |
+| omSupply Permissions |                       | Cold chain API access               | Can access the Cold Chain API                                                                                |
+| Permissions (3)      | Assets                | View assets                         | Can view Assets                                                                                              |
```

Comment:
can you re-order these, so that we have the permissions grouped in order 🙏 

- Permissions
- Permissions (2)
- Permissions (3)
- omSupply Permissions